surfstat/python/SurfStatResels.py

```diff
@@ -31,7 +31,7 @@ def py_SurfStatResels(slm, mask=None):
     resels : a 2D numpy array of shape (1, (D+1)).
         Array of 0,...,D dimensional resels of the mask, EC of the mask 
         if slm['resl'] is not given.
-    reselspvert : a 1D numpy array of shape (v,).
+    reselspvert : a 2D numpy array of shape (1,v).
```

Comment:
This change doesn't make much sense to me. Our goal should be to have a SurfStat python implementation that has the same functionality within the limitations of each language's architectures - trying to exactly match across languages will cause more confusion for end users than be helpful. 

Having a singleton dimension as the first dimension matches MATLAB (since MATLAB doesn't support 1D arrays) but it doesn't make much sense within a Pythonic framework.